views/form.php

```diff
@@ -177,13 +152,13 @@
 	</div>
 	<div class="row">
 		<div class="col-md-12">
-			<span id="opencnam_account_sid-help" class="help-block fpbx-help-block"><?php echo _("Your OpenCNAM Account SID. This can be found on your OpenCNAM dashboard page: https://www.opencnam.com/dashboard")?></span>
+			<span id="opencnam_account_sid-help" class="help-block fpbx-help-block"><?php echo _("Your OpenCNAM Account SID. This can be found on your OpenCNAM dashboard page:")?> <a href="https://www.opencnam.com/dashboard" target="_blank">https://www.opencnam.com/dashboard</a></span>
```

Comment:
This needs to be an sprintf(%s) for rtl languages. Will fix on my end. Making a note.